crates/cast/src/helpers/response_structs.rs

```diff
@@ -29,3 +30,9 @@ pub struct AccountCreateResponse {
     pub max_fee: FieldElement,
     pub add_profile: String,
 }
+
+#[derive(Serialize)]
+pub struct MulticallNewResponse {
+    pub path: Utf8PathBuf,
+    pub body: String,
```

Comment:
Would `content` be a better name?

Reply:
Sure, I like it :+1:. Changed in newest commit.